shell/platform/darwin/graphics/FlutterDarwinExternalTextureMetal.mm

```diff
@@ -133,8 +133,11 @@ - (void)onTextureUnregistered {
   if (_pixelFormat == kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange ||
       _pixelFormat == kCVPixelFormatType_420YpCbCr8BiPlanarFullRange) {
     image = [self wrapNV12ExternalPixelBuffer:pixelBuffer context:context];
-  } else {
+  } else if (_pixelFormat == kCVPixelFormatType_32BGRA) {
     image = [self wrapRGBAExternalPixelBuffer:pixelBuffer context:context];
```

Comment:
This method is incorrectly named. Perhaps rename to `wrapBGRAExternalPixelBuffer:context:` too? Internally, it does assume BGRA as well.

Reply:
Done. Note that `[FlutterDarwinExternalTextureSkImageWrapper: wrapRGBATexture:grContext:width:height:]` also has RGBA in its name. I'm not sure renaming all of them to use BGRA is a good idea, but I'm open to suggestions.

Reply:
If its not already a public method and the name is more fitting, go for it!

Reply:
I was considering renaming the `wrapRGBATexture` method, but I think it is better to keep the method name as it is.

[Link1](https://github.com/flutter/engine/blob/8578edf9c9393471ca9eab18e9154f0e6066dcb6/shell/platform/embedder/embedder.h#L602)                                                                                                                      [Link2](https://github.com/flutter/engine/blob/8578edf9c9393471ca9eab18e9154f0e6066dcb6/shell/platform/embedder/embedder_external_texture_metal.mm#L73)

We have `kYUVA` and `kRGBA` as the two possible values for `FlutterMetalExternalTexturePixelFormat`, and the function I mentioned above is used along with the enum, `FlutterMetalExternalTexturePixelFormat`. So it sounds like renaming would be more confusing.